xhr.bs

```diff
@@ -1122,10 +1124,10 @@ exception <var>exception</var> are:
   </ol>
 
  <li><p><a>Fire a progress event</a> named
- <var>event</var> with 0 and 0.
+ <var>event</var> at [=this=] with 0 and 0.
```

Comment:
Since a corresponding event is fired at the request upload object in the previous step, I'm fairly certain the intended target here is the request object.